src/main/java/com/liferay/ide/idea/ui/modules/LiferayWorkspaceBuilder.java

```diff
@@ -64,23 +70,49 @@ public ModuleType getModuleType() {
 	public ModuleWizardStep modifySettingsStep(@NotNull SettingsStep settingsStep) {
 		JComboBox liferayVersionComboBox = new ComboBox();
 
-		liferayVersionComboBox.addItem("7.0");
-		liferayVersionComboBox.addItem("7.1");
+		Stream.of(
+			WorkspaceConstants.LIFERAY_VERSIONS
+		).forEach(
+			liferayVersion -> liferayVersionComboBox.addItem(liferayVersion)
+		);
+
+		settingsStep.addSettingsField("Liferay version:", liferayVersionComboBox);
+
+		JComboBox targetPlatformComboBox = new ComboBox();
+
+		Stream.of(
+			WorkspaceConstants.TARGET_PLATFORM_VERSIONS_7_0
+		).forEach(
+			targetPlatform -> targetPlatformComboBox.addItem(targetPlatform)
+		);
+
+		Stream.of(
+			WorkspaceConstants.TARGET_PLATFORM_VERSIONS_7_1
+		).forEach(
+			targetPlatform -> targetPlatformComboBox.addItem(targetPlatform)
+		);
+
+		targetPlatformComboBox.addActionListener(
+			e -> _targetPlatform = StringUtil.toString(targetPlatformComboBox.getSelectedItem()));
+
+		if ("LiferayGradleWorkspce".equals(_liferayProjectType)) {
+			settingsStep.addSettingsField("Target platform:", targetPlatformComboBox);
+		}
 
 		liferayVersionComboBox.addActionListener(
-			new ActionListener() {
+			e -> {
+				String selected = StringUtil.toString(liferayVersionComboBox.getSelectedItem());
 
-				@Override
-				public void actionPerformed(ActionEvent e) {
-					Object selectedVersion = liferayVersionComboBox.getSelectedItem();
+				_liferayVersion = selected;
 
-					_liferayVersion = selectedVersion.toString();
+				if ("7.0".equals(selected)) {
```

Comment:
This doesn't scale well... we may have more versions in the future, 7.1.1 or 7.1.10.1 or etc.  Is there something we can do better?

Reply:
For now maybe not, but in the future we have to consider this, both here and in Eclipse plugin.